logger.js

```diff
@@ -160,14 +161,17 @@ logger.log = function (level, data) {
 logger.log_respond = function (retval, msg, data) {
     // any other return code is irrelevant
     if (retval !== constants.cont) { return false; }
-
+    let timestamp_string = '';
+    if (logger.timestamps) {
+        timestamp_string = `${new Date().toISOString()} `;
```

Comment:
Don't you have a "why are my log timestamps N days ago" problem going on here?

Reply:
Nope, but there's another bug actually, set_timestamps is being set twice, first by the .ini, second by the value file.

value seems to always be used no matter what the ini says.

I'm working on that now